apps/web/app/(auth)/_component/register-form.tsx

```diff
@@ -0,0 +1,88 @@
+"use client";
+
+import type { SubmitHandler } from "react-hook-form";
+import { useForm } from "react-hook-form";
```

Comment:
```suggestion
import { useForm, type SubmitHandler } from "react-hook-form";
```
요렇게 하는것도 좋을 것 같아요! 일부러 분리시킨 걸까요?